app/models/flexible_criterion.rb

```diff
@@ -29,7 +29,7 @@ class FlexibleCriterion < Criterion
 
   validates_presence_of :max_mark
   validates_numericality_of :max_mark,
-                            message: 'must be a number greater than 0.0',
+                            message: I18n.t('flexible_criteria.errors.messages.input_number'),
```

Comment:
Use the same key for both rubric and flexible criteria. Put it under `criteria.errors` or something.